### PR TITLE
cleanup readme a bit

### PR DESCRIPTION
**Pull request by grosser**
@nevans

### CODE REVIEW

**Comment by grosser** [comment]
also updated the changelog

**Comment by grosser** [comment]
rebased

File: README.md [hunk 1]
```diff
@@ -159,10 +165,10 @@ value stored in Redis, you could do something like:
...
 ```ruby
 task resque:pool:setup do
-Resque::Pool.config_loader = lambda {|env|
-  worker_count = Redis.current.get("pool_workers_#{env}").to_i
-  {"queueA,queueB" => worker_count }
-}
+  Resque::Pool.config_loader = lambda do |env|
+    worker_count = Redis.current.get("pool_workers_#{env}").to_i
+    {"queueA,queueB" => worker_count }
+  end
```
Comment (nevans) [inline README.md]: I've always preferred the Weirich Semantic Rule for blocks: https://github.com/bbatsov/ruby-style-guide/issues/162

But I'll admit I seem to be in the minority.  ;)

**Comment by grosser** [comment]
last one ... release ? :)

**Pull request merged**